libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -455,26 +455,27 @@ def validate_environment(cls, values: Dict) -> Dict:
     @property
     def _default_params(self) -> Dict[str, Any]:
         """Get the default parameters for calling OpenAI API."""
-        params = {
-            "model": self.model_name,
-            "stream": self.streaming,
-            "n": self.n,
-            "temperature": self.temperature,
+        exclude_if_none = {
             "presence_penalty": self.presence_penalty,
             "frequency_penalty": self.frequency_penalty,
             "seed": self.seed,
             "top_p": self.top_p,
             "logprobs": self.logprobs,
             "top_logprobs": self.top_logprobs,
             "logit_bias": self.logit_bias,
+            "stop": self.stop or None,  # also exclude empty list for this
+            "max_tokens": self.max_tokens,
+            "extra_body": self.extra_body,
+        }
+
+        params = {
+            "model": self.model_name,
+            "stream": self.streaming,
+            "n": self.n,
+            "temperature": self.temperature,
+            **{k: v for k, v in exclude_if_none.items() if v is not None},
             **self.model_kwargs,
         }
-        if self.max_tokens is not None:
-            params["max_tokens"] = self.max_tokens
-        if self.stop:
```

Comment:
nit: stop is excluded if its falsey. is default value for stop None or empty list?

Reply:
I think this was handled by the `self.stop or None` above